src/services/stellar/client.test.ts

```
@@ -0,0 +1,95 @@
+
+import { StellarClient } from "./client";
+import { logger } from "../../config/logger";
+import { config } from "../../config/env";
+
+const VALID_SECRET = "SCZANGBA5YHTNYVVVVCG2XTIBQ4SKWDJXG3G5C2JKKQLOOOQ2K3X7LKP";
+
+beforeEach(() => {
+  jest.clearAllMocks();
+  mockFromSecret.mockImplementation((secret: string) => {
+    if (secret === VALID_SECRET) {
+      return { publicKey: () => "GDZRKO6ZP4C5DQ4V6T6P6R4H7ZV7R56L5V6P6R4H7ZV7R56L5V6P6" };
```

ESLint

[error] Replace `·publicKey:·()·=>·"GDZRKO6ZP4C5DQ4V6T6P6R4H7ZV7R56L5V6P6R4H7ZV7R56L5V6P6"` with `⏎········publicKey:·()·=>⏎··········"GDZRKO6ZP4C5DQ4V6T6P6R4H7ZV7R56L5V6P6R4H7ZV7R56L5V6P6",⏎·····`

(prettier/prettier)